source/install/smtp-email-setup.rst

```diff
@@ -5,6 +5,8 @@ SMTP Email Setup
 
 To run in production, Mattermost requires SMTP email to be enabled for email notifications and password reset for systems using email-based authentication.
 
+If you have any problems installing, see the `troubleshooting guide <https://www.mattermost.org/troubleshoot/>`__. To submit an improvement or correction, click Edit at the top of this page.
```

Comment:
This should probably be "setting up" rather than "installing". Also, see https://docs.mattermost.com/install/install-ubuntu-1604.html for an example of this note.